articles/azure/azs-how-resize-disks.md

```diff
@@ -0,0 +1,37 @@
+---
+title: How to resize virtual machine disks
+description: Learn how to resize virtual machine disks
+services: azure-stack
+author: David Woffendin 
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Disk Management
+toc_sub3:
+toc_sub4:
+toc_title: How to resize virtual machine disks
+toc_fullpath: Users/How To/azs-how-resize-disks.md
+toc_mdlink: azs-how-resize-disks.md
+---
+
+# How to resize virtual machine disks
+
+In this article we will cover the process of resizing disks in the Azure portal. This will be needed if the VM requires more storage space as you can increase current disk capacity rather than adding more disks.
+
+## Steps to changing disk size via portal
+
+1. Power down or detach disk from VM.
+  ![Detach disks](images/azs-browser-detach-disk.png)
+2. Navigate to the resource group.
+3. Select disk.
+4. Alter the disk size (Note the max size for a disk is 1023gb for a single disk).
+  ![Detach disks](images/azs-browser-change-disk.png)
+5. Save changes.
+6. Power back on the VM or reattach the disk.
+  ![Detach disks](images/azs-browser-attach-disk.png)
+
+These steps will work for both non managed and managed VM's.
```

Comment:
I'd move this to the top, just before the numbered listed starts.

And get rid of the apostrophe in VMs